src/components/Dialogs/FittingDialog/FittingDialogComponent.scss

```diff
@@ -66,9 +66,9 @@
     }
 
     .bp3-label {
-        display: flex;
```

Comment:
I suggest we keep `display: flex;` and replace `text-align: right;` with `justify-content: end;`. This can keep the labels are on the same line.

Reply:
done

Reply:
It seems that ```justify-content: end;``` is not allowed for CI, and ```display: flex-end;``` doesn't work.

Reply:
replace ```justify-content: end;``` with ```justify-content: flex-end;```, bring back ```display: flex;``` and remove ```text-align: right;```